assignment/scripts/function-practice.js

```diff
@@ -15,30 +15,34 @@ console.log('Test - should say "Hello World!"', hello());
 // 2. Function to return an personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
 function helloName( name ) {
-  return;
+  return `Hello, ${name}!`;
 }
 // Remember to call the function to test
-
+console.log('Should say "Hello, Maya!":', helloName('Maya'));
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
-  // return firstNumber + secondNumber;
+function addNumbers( firstNumber, secondNumber ) {
+  return firstNumber + secondNumber;
 }
 
+console.log('Should say 8:', addNumbers(3, 5));
+console.log('Should say -8:', addNumbers(-3, -5));
 
-// 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
 
+// 4. Function to multiply three numbers & return the result
+function multiplyThree(firstNum, secondNum, thirdNum ){
+  return firstNum*secondNum*thirdNum;
```

Comment:
Spaces are great for readability.

```suggestion
  return firstNum * secondNum * thirdNum;
```